Dockerfile

```diff
@@ -129,8 +129,8 @@ COPY build.d common/build.d
 COPY conf.d common/conf.d
 COPY entrypoint.d common/entrypoint.d
 RUN mkdir -p auto/addons custom/src/private \
-    && ln /usr/local/bin/direxec.sh common/entrypoint.sh \
-    && ln /usr/local/bin/direxec.sh common/build.sh \
+    && ln /usr/local/bin/direxec.py common/entrypoint \
+    && ln /usr/local/bin/direxec.py common/build \
```

Comment:
For me it's a matter of how things look in Git. When I'm reviewing things, I pay attention to whether something is checked in with executable permissions, because they basically should never. No extension to me tells me that the file is known to be an executable and should have the permissions as such. An extension tells me there shouldn't be an executable bit. Maybe it's an oversimplification, but this is my main reason for pushing back.